buildpacktoml_test.go

```diff
@@ -286,32 +288,107 @@ func testBuildpackTOML(t *testing.T, when spec.G, it spec.S) {
 					}
 				})
 
-				it("should include processes", func() {
-					h.Mkfile(t,
-						`[[processes]]`+"\n"+
-							`type = "some-type"`+"\n"+
-							`command = "some-cmd"`+"\n"+
+				when("processes", func() {
+					when("platform API < 0.6", func() {
+						it.Before(func() {
+							config.PlatformAPI = "0.5"
+						})
+						it("should include processes and use the default value that is set", func() {
+							h.Mkfile(t,
+								`[[processes]]`+"\n"+
+									`type = "some-type"`+"\n"+
+									`command = "some-cmd"`+"\n"+
+									`default = true`+"\n"+
+									`[[processes]]`+"\n"+
+									`type = "web"`+"\n"+
+									`command = "other-cmd"`+"\n",
+								// default is false and therefore doesn't appear
```

Comment:
Having `default= = false` in the TOML would also be fine, if we want to do it that way because it's clearer.

Reply:
We thought that it's fine to not write `default` to the TOML when its value is `false`, but I don't have strong opinion on that.
If you think it'll be clearer to write it anyway, we can make this change.

Reply:
I'm okay leaving it this way!